vms/platformvm/txs/fee/static_config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fee
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/vms/platformvm/upgrade"
+)
+
+type StaticConfig struct {
```

Comment:
This used to be in config package and I moved it here, following @joshua-kim suggestions to simplify platformvm config.
A DynamicConfig will be added in next PRs, hence the Static suffix.
With this change, everything fee related is in the fee package; only initialization of fee config attributes is done via platformvm config.
This may change in the future, but it's not strictly related to dynamic fees, so postponed the change